app/assets/javascripts/widgets/brochure_masthead.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function() {
+  $(".offcanvas-menu-trigger a").on("click", function(){
+    $("#js-cu-off-canvas-nav-trigger").trigger("click");
+    return false;
```

Comment:
Ok. I looked again at the design and it makes more sense. Maybe add a comment noting this is for the VIEW MENU button in masthead. Also, I like `brochure.js` suggestion.